scripts/init.sh

```diff
@@ -5,12 +5,8 @@ set -e
 echo "*** Initializing WASM build environment"
 
 if [ -z $CI_PROJECT_NAME ] ; then
-   #rustup update nightly
+   rustup update nightly
    rustup update stable
 fi
 
-rustup target add wasm32-unknown-unknown --toolchain $RUST_TOOLCHAIN
-
-# Install wasm-gc. It's useful for stripping slimming down wasm binaries.
-command -v wasm-gc || \
```

Comment:
I noticed that when I first built the project I needed this wasm-gc , is there any reason why we are removing it?

Reply:
True, the substrate team changed that requirement, the wasm optimization is happening within the wasm build stage now.